Flutter 2 update: Build fail due to flutter_svg plugin

With the flutter 2 update, many beta changes were pushed to the stable channel of flutter and due to this many things are changing and one of the things that became available is the latest version of flutter_svg plugin (an older version of the same plugin is being used in the app due to the reasons discussed in #102). 

### Console error (Build fail)
```bash
/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_svg-0.18.0/lib/src/picture_provider.dart:50:59: Error: No named parameter with the name 'nullOk'.
        context != null ? Localizations.localeOf(context, nullOk: true) : null,
                                                          ^^^^^^
/C:/src/flutter/packages/flutter/lib/src/widgets/localizations.dart:413:17: Context: Found this candidate, but the arguments don't match.
  static Locale localeOf(BuildContext context) {
                ^^^^^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\src\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 991

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\src\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 48s
Exception: Gradle task assembleDebug failed with exit code 1
```

Right now people who have upgraded to flutter 2 will get this error on building the app. I would like to fix this.

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -53,7 +53,7 @@ dependencies:
   hive: ^1.4.4+1
   hive_flutter: ^0.3.1
   # open_file: ^3.0.3
-  flutter_svg: 0.18.0
+  flutter_svg: ^0.19.3
 
 dev_dependencies:
   flutter_test:

